### DISCUSSION TITLE
Why does the with-tailwind example import react into the button?

### DISCUSSION THREAD

**Discussion by mrv777** [vercel/turborepo#3016] [Help] [answered]
In the example [with-tailwind](https://github.com/vercel/turbo/tree/main/examples/with-tailwind), the share Button component imports React.  Why does it do that?  It doesn't seem to use react there
https://github.com/vercel/turbo/blob/main/examples/with-tailwind/packages/ui/src/Button.tsx#L1

Same Question about the index.tsx file there, why export React?
https://github.com/vercel/turbo/blob/main/examples/with-tailwind/packages/ui/src/index.tsx

Thank you

**Comment by theaungmyatmoe** [comment, marked as answer]
`.jsx` is just a sugar for IDEs and DX. So, you can assume it as `.js`. To use jsx in js you also need to import `React` to compile JSX syntax to something like magic lol.
